x-pack/plugins/rollup/public/crud_app/services/api_errors.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { fatalError, toastNotifications } from 'ui/notify';
+
+function createToast(error, errorTitle) {
```

Comment:
nitpick: I thought this would create & return the toast object itself, but it's more formatting the error data so that it can be consumed by `toastNotifications`. maybe a better name would be `getToastFormat`?

Reply:
Renamed to `createToastConfig`.